### PR TITLE
[MXAPPS-1020] Clean up some Sphinx warnings.

### PR DESCRIPTION
**Pull request by vishaalkapoor**
## Description ##
* Correct some markup that is causing Sphinx warnings. 
* Remove illegal line-break and elipsis
* Remove isinf, isfinite, isnan as they are not in the symbol.contrib package.
* Add whitespace after docstring in .cc file.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
NCCL)
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by vishaalkapoor** [comment]
@aaronmarkham Please take a look. I removed the Scala files that were causing warnings (we talked about offline) as you've edited those.

File: docs/gluon/index.md [hunk 1]
```diff
@@ -82,7 +82,6 @@ def forward(self, F, inputs, tree):
                         for child in tree.children]
     #Recursively builds the neural network based on each input sentence’s
     #syntactic structure during the model definition and training process
-    …
```
Comment (aaronmarkham) [inline docs/gluon/index.md]: Was this breaking something or is your editor linting things?

File: src/operator/contrib/dgl_graph.cc [hunk 2]
```diff
@@ -795,7 +795,8 @@ uniform probability.
 
   out[2]
   [0 0 0 0 0]
-<NDArray 5 @cpu(0)>
+  <NDArray 5 @cpu(0)>
```
Comment (aaronmarkham) [inline src/operator/contrib/dgl_graph.cc]: I found this to be a problem as well.

**Comment by aaronmarkham** [comment]
Ok, we're good to remove those entries on the symbol page.

File: docs/faq/develop_and_hack.md [hunk 3]
```diff
@@ -5,4 +5,4 @@
 
 # Other Resources
 - [MXNet System Architecture Overview](http://mxnet.io/architecture/overview.html)
-- [Contributor Guidelines](http://mxnet.io/community/contribute.html)
\ No newline at end of file
+- [Contributor Guidelines](http://mxnet.io/community/contribute.html)
```
[the remark this answers is not in the document]
Comment (vishaalkapoor) [reply docs/faq/develop_and_hack.md]: This just removes an invisible character that's causing a . But I'll update the two links to be relative.

File: docs/gluon/index.md (hunk 1 above)
Comment (vishaalkapoor) [reply docs/gluon/index.md]: This was causing an warning, and when removed, I was able to remove that warning, but now I'm unable to reproduce so I've added it back in. However, this does look like it shouldn't be there. It's a UTF-8 ellipsis symbol.

File: src/operator/contrib/dgl_graph.cc (hunk 2 above)
Comment (vishaalkapoor) [reply src/operator/contrib/dgl_graph.cc]: Nice

**Comment by vishaalkapoor** [comment]
@aaronmarkham Updated as per comments. thanks!

**Pull request merged**